open-metadata-implementation/adapters/governance-engines-plugins/gaian-ranger-plugin/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+#SPDX-License-Identifier: Apache-2.0
+#Copyright Contributors to the ODPi Egeria project.
+ranger.plugin.gaian.policy.rest.url=http://localhost:6080
```

Comment:
why do we need a new property /file-- the ranger plugin already has to be configured to point to the ranger server in one of the existing xml files

Reply:
this property is used by the plugin itself in order to call the services for getting the user details and groups

Reply:
In the last commit, I removed the application properties file and the Ranger URL is fetched from XML using RangerConfiguration. 
For REST call authentication we might discuss more details. We can use service-def or something similar.